src/pages/ProjectSettings.tsx

```diff
@@ -116,9 +118,22 @@ export const ProjectSettings: React.FC = () => {
     ? projectType === ProjectType.Project || repoData
     : repoData;
 
+  const { repoSettings } = repoData || {};
+  const { projectRef } = repoSettings || {};
+  const { owner, repo } = projectRef || {};
+
+  const repoDisplay = owner && repo ? `${owner}/${repo}` : identifier;
+
   return (
     <ProjectSettingsProvider>
       <SideNav aria-label="Project Settings">
+        <DropdownContainer>
+          <ProjectSelect
+            selectedProjectIdentifier={repoDisplay}
+            getRoute={getProjectSettingsRoute}
+            isProjectSettingsPage
+          />
+        </DropdownContainer>
         <SideNavGroup header="Project" />
```

Comment:
These sidebar elements can also go in the same side nav group:

```js
<SideNavGroup>
  <DropdownContainer>
    <ProjectSelect ... />
  </DropdownContainer
  <CreateProjectModal ... />
</SideNavGroup>

Reply:
It matches the design better when it's outside of it
<img width="190" alt="image" src="https://user-images.githubusercontent.com/13104717/157899057-e2193ada-3e26-4839-98e1-72898f3c4450.png">